File: UTime/UTime/ViewController.m
//
//  ViewController.m
//  UTime
//
//  Created by AdrianHsu on 2015/5/7.
//  Copyright (c) 2015年 AdrianHsu. All rights reserved.
//

#import "ViewController.h"
#import <FBSDKCoreKit/FBSDKCoreKit.h>
#import <FBSDKLoginKit/FBSDKLoginKit.h>

@interface ViewController ()

@property (weak, nonatomic) IBOutlet UIPickerView *Countdown;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    FBSDKLoginButton *loginButton = [[FBSDKLoginButton alloc] init];
    loginButton.center = CGPointMake(160, 480);
    //CGRectMake(<#CGFloat x#>, <#CGFloat y#>, <#CGFloat width#>, <#CGFloat height#>)
    [self.view addSubview:loginButton];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: iOS_week5_onclass/iOS_week5_onclass/ViewController.swift
//
//  ViewController.swift
//  iOS_week5_onclass
//
//  Created by AdrianHsu on 2015/4/28.
//  Copyright (c) 2015年 AdrianHsu. All rights reserved.
//

//TASK: Autolayout

import UIKit

class ViewController: UIViewController {

    
    @IBOutlet weak var testView:UIView!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
        self.testView.backgroundColor = UIColor.redColor()
        
        
        
//        self.testView.layer.backgroundColor = UIColor.redColor().CGColor
        
    }
    

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
}

